core/frontend/src/test/scala/io/udash/bindings/TagsBindingTest.scala

```diff
@@ -1904,4 +1904,48 @@ class TagsBindingTest extends UdashFrontendTest with Bindings { bindings: Bindin
       testDiv.innerHTML shouldBe empty
     }
   }
+
+  "InlineStyleOps" should {
+    "allow reactive inlined style bind" in {
+      val styleProperty = Property("red")
+      val pixelStyleProperty = Property("10px")
+      val testDiv = div(
+        backgroundColor.bind(styleProperty),
+        width.bind(pixelStyleProperty)
+      ).render
+      testDiv.style.getPropertyValue("background-color") should ===("red")
+      testDiv.style.getPropertyValue("width") should ===("10px")
+      styleProperty.set("black")
+      pixelStyleProperty.set("100px")
+      testDiv.style.getPropertyValue("background-color") should ===("black")
+      testDiv.style.getPropertyValue("width") should ===("100px")
+      styleProperty.set(null)
+      pixelStyleProperty.set(null)
+      testDiv.style.getPropertyValue("background-color") should ===("")
+      testDiv.style.getPropertyValue("width") should ===("")
+      styleProperty.set("blue")
+      pixelStyleProperty.set("2rem")
+      testDiv.style.getPropertyValue("background-color") should ===("blue")
+      testDiv.style.getPropertyValue("width") should ===("2rem")
```

Comment:
Important part of the `Binding` interface is the `kill` method. We should test if it clears the listeners properly. 
For example:
```scala
val bindStyle = backgroundColor.bind(styleProperty)
val bindPixelStyle = width.bind(pixelStyleProperty)
val testDiv = div(bindStyle, bindPixelStyle).render
...
styleProperty.listenersCount() should be(1)
pixelStyleProperty.listenersCount() should be(1)

bindStyle.kill()
styleProperty.listenersCount() should be(0)
pixelStyleProperty.listenersCount() should be(1)

bindPixelStyle.kill()
styleProperty.listenersCount() should be(0)
pixelStyleProperty.listenersCount() should be(0)
```